rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -194,8 +145,16 @@ private void init(String url, String login, String password, boolean insecure) t
         this.url = url;
         this.login = login;
         this.password = password;
+        this.initialized = true;
```

Comment:
I did it like this at first, but the renewSession method makes use of the restApi() method which now checks that initialization is done. 

In addition, the scheduler client is not thread safe.